src/services/milestones/milestones.hooks.js

```diff
@@ -77,7 +78,7 @@ const getApprovedKeys = (milestone, data, user) => {
         }
         logger.info(`Rejecting proposed milestone with id: ${milestone._id} by: ${user.address}`);
 
-        return ['status'];
+        return ['status', 'mined', 'message'];
```

Comment:
mined doesn't need to be here, this is a proposed milestone which isn't on chain